rados/rados.go

```diff
@@ -81,6 +80,11 @@ func (c *RadosConn) newRadosConn() (*rados.Conn, error) {
 		return nil, fmt.Errorf("error setting rados_mon_op_timeout: %s", err)
 	}
 
+	err = conn.SetConfigOption("client_mount_timeout", tv)
+	if err != nil {
+		return nil, fmt.Errorf("error setting client_mount_timeout: %s", err)
+	}
+
 	ch := make(chan error, 1)
 	go func(conn *rados.Conn) {
```

Comment:
It would be good to add a comment here around why we need to have the goroutine timeout wrapper despite setting `client_mount_timeout` above.

Reply:
Good idea